src/renderer/services/eth2/client/prysm/mapProduceBlockDataToPrysmProduceBlock.ts

```diff
@@ -0,0 +1,117 @@
+import {
+    AttestationSlashingAttestation,
+    Attestation,
+    AttestationSlashing,
+    AttestationData,
+    Deposit,
+    ProposerSlashing,
+    SignedBeaconBlockData,
+    SignedHeader,
+    VoluntaryExit,
+} from "./map.types";
+import {
+    Attestation as PrysmAttestation,
+    AttestationData as PrysmAttestationData,
+    AttesterSlashing as PrysmAttesterSlashing,
+    Deposit as PrysmDeposit,
+    IndexedAttestation as PrysmIndexedAttestation,
+    ProposerSlashing as PrysmProposerSlashing,
+    SignedBeaconBlockHeader as PrysmSignedBeaconBlockHeader,
+    SignedVoluntaryExit as PrysmSignedVoluntaryExit,
+    SignedBeaconBlock,
+} from "./types";
+import {hexToBase64} from "./utils";
+
+/* eslint-disable camelcase */
+/* eslint-disable @typescript-eslint/camelcase */
+
+export const mapProduceBlockDataToPrysmProduceBlock = (data: SignedBeaconBlockData): SignedBeaconBlock => ({
```

Comment:
why do you serialize to json before converting. Wouldn't it be easier to convert object directly to this prysm format?

Reply:
this is the simplest solution but not the best, you got a hex string that needs to be converted into the base64, don't need to bother with lodestar that complicates typings as with them need to cast types or use config to convert it